loudness_pcm.c

```diff
@@ -478,18 +488,8 @@ static snd_pcm_sframes_t transfer_callback(
     return size;
 }
 
-static int init_callback(snd_pcm_extplug_t *ext)
-{
+static int init_callback(snd_pcm_extplug_t *ext) {
     struct context *context = (struct context *)ext->private_data;
```

Comment:
Since this is now only used when threading is enabled, it should be moved inside the `#ifdef WITH_THREADS` below.